### ISSUE TITLE
Publish our browser compatibility Open Sauce (Sauce Labs) page

### ISSUE THREAD

**Issue by mattheworiordan** [ably/ably-js#16]
Once the integration with SauceLabs is complete, publish a link to the page from the module readme.

**Comment by mattheworiordan** [comment]
https://github.com/ably/ably-js/issues/11

**Comment by SimonWoolf** [comment]
Given the problems we continue to have with saucelabs (test runs [failing](https://ci.ably.io/job/ably-js-browsers-min/753/console) to [connect to saucelabs](https://ci.ably.io/job/ably-js-browsers-min/751/console) at all, plus the general flakeyness we've seen (eg https://github.com/ably/ably-js/pull/229) if they do connect), I doubt this'll happen any time soon.

cf https://github.com/theintern/intern/issues/394

**Comment by funkyboy** [comment]
Closing this. Will be superseded by #457
